contents/tutorials/bounce-rate.md

```diff
@@ -1,102 +1,136 @@
 ---
 title: How to calculate bounce rate
-date: 2023-06-27
+date: 2024-12-11
 author:
   - ian-vanagas
+  - bijan-boustani
 showTitle: true
 sidebar: Docs
 tags:
   - insights
   - hogql
   - product analytics
+  - web analytics
 ---
 
-Bounce rate is the percentage of users who leave your page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors.  
+Bounce rate is the percentage of users who leave a page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors. This tutorial shows you how to calculate bounce rate in PostHog.
```

Comment:
> Bounce rate is the percentage of users who leave a page immediately after visiting.

Is this accurate? I thought bounce rate was users who land on a page and do nothing?

Looks like this is the [definition they use in GA4](https://support.google.com/analytics/answer/12195621?hl=en). I actually vaguely remember bringing this up when @ivanagas first wrote this tutorial. I don't remember what the answer was, though

Reply:
Right. So that means the definition here is incorrect of leave a page "immediately" is incorrect

Reply:
Agreed that the intro was a little confusing when contrasted with how we define bounce rate. I went ahead and rewrote the intro here and I think it's much clearer now:
https://github.com/PostHog/posthog.com/pull/10106/files#diff-053c49ad5fdf99aeafb37ce839ecd20f3037aee42e833ed000974052a3e075aeR16